conformance/eexp/binary/argument_encoding.ion

```diff
@@ -0,0 +1,625 @@
+                           (binary "61 00 F0")
+                           (binary "62 00 00 F0")
+                           (produces 0))
+                     (each "and contains multiple values"
+                           // Int 0 with various amounts of overpadding, Float 0
+                           (binary "60 6A F0")
+                           (binary "61 00 6A F0")
+                           (binary "62 00 00 6A F0")
+                           (produces 0 0e0)))))
+
+//////////////////////////////////////////////////////////////////////////////////////////////
+//                 Single tagless, fixed-sized, single-byte parameter                       //
+//////////////////////////////////////////////////////////////////////////////////////////////
+
+(ion_1_1 "a macro with a tagless, single-byte, required parameter"
+         (mactab (macro null (uint8::x) (%x)))
+         (then "when invoked with zero arguments"
+               (binary "00")
+               (signals "Unexpected EOF"))
+         (then "when invoked with a single argument"
+               (binary "00 01")
+               (produces 1)))
+
+(ion_1_1 "a macro with a tagless, single-byte, zero-to-one parameter"
+         (mactab (macro null (uint8::x?) (%x)))
+         (binary "00")
+         (then "when invoked with no presence bitmap, raises an error" (signals "Unexpected EOF"))
+         (then "when invoked with no arguments"
+               (binary "00") // Presence bitmap indicating no arguments
+               (produces))
+         (then "when invoked with a single argument"
+               (binary "01") // Presence bitmap indicating 1 argument
+               (binary "01") // u8 `1`
+               (produces 1))
+         (then "when invoked with an expression group"
+               (binary "02") // Presence bitmap indicating expression group
+               (then "that is length prefixed"
+                     // L=0 is the escape for delimited expression groups, so there is no empty case here
+                     (then "and contains one value"
+                           // L=1 bytes, u8 `1`
+                           (binary "03 01")
+                           (produces 1))
+                     (then "and contains multiple values"
+                           // L=2 bytes, u8 `1`, u8 `2`
+                           (binary "05 01 02")
+                           (signals "invalid argument")))
+               (then "that is delimited"
+                     (binary "01")
+                     (then "and empty"
+                           (binary "01")
+                           (produces))
+                     (then "and contains one value"
+                           // L=1 bytes, u8 `1`, delimited end
+                           (binary "03 01 01")
+                           (produces 1))
+                     (each "and contains multiple values"
+                           // L=2 bytes, u16 `1`, u16 `2`, delimited end
```

Comment:
```suggestion
                           // L=2 bytes, u8 `1`, u8 `2`, delimited end
```
Or, to use Ion 1.1 naming rather than Rust's:
```suggestion
                           // L=2 bytes, uint8 `1`, uint8 `2`, delimited end
```